sdklab/longhaultests/package.json

```diff
@@ -9,7 +9,7 @@
     "lint": "tslint --project . -c ../../tslint.json",
     "build": "tsc",
     "setup_links": "npm link azure-iot-common azure-iot-device azure-iot-device-amqp azure-iot-device-mqtt azure-iot-device-http azure-iothub",
-    "start": "node lib/iothub_longhaul.js"
+    "start": "node dist/iothub_longhaul.js"
```

Comment:
Ignore everything in sdklab for now.  I'm not planning to pull any of this right now.